src/Microsoft.AspNetCore.SignalR.Common/Internal/Protocol/CompletionMessage.cs

```diff
@@ -10,31 +10,50 @@ public class CompletionMessage : HubMessage
         public string Error { get; }
         public object Result { get; }
         public bool HasResult { get; }
+        public bool IsStreamingCompletion { get; }
```

Comment:
The question is whether to use this flag or create a separate `StreamCompletion` message which will not have Result and `HasResult` properties (possibly a base completion class)

Reply:
I think it should be a new class

Reply:
I am adding a new class in TypeScript client to how it looks like.